### PR TITLE
[CRON TASKS] added loggers

### PR DESCRIPTION
**Pull request by RuslanHamidullin**
In accordance with (trello story)[https://trello.com/c/Iz4TuGmg/115-what-is-the-time-day-we-send-data-to-the-performance-platform]

### CODE REVIEW

File: app/services/notifiers/email_notification_service.rb [hunk 1]
```diff
@@ -98,4 +102,12 @@ def winners_press_release_comments_request(award_year)
       end
     end
   end
+
+  private
```
Comment (matthewford) [inline app/services/notifiers/email_notification_service.rb]: Useless `private` access modifier.

File: app/services/notifiers/email_notification_service.rb [hunk 2]
```diff
@@ -98,4 +102,12 @@ def winners_press_release_comments_request(award_year)
...
+
+  class << self
+    def log_this(message)
+      p "[EmailNotificationService] #{Time.zone.now} #{message}"
```
Comment (matthewford) [inline app/services/notifiers/email_notification_service.rb]: Do not write to stdout. Use Rails' logger if you want to log.

File: app/services/performance_platform_service.rb [hunk 3]
```diff
@@ -199,4 +203,12 @@ def self.md5(string)
 
     md5.hexdigest
   end
+
+  private
```
Comment (matthewford) [inline app/services/performance_platform_service.rb]: Useless `private` access modifier.

File: app/services/performance_platform_service.rb [hunk 4]
```diff
@@ -199,4 +203,12 @@ def self.md5(string)
...
+
+  class << self
+    def log_this(message)
+      p "[PerformancePlatformService] #{Time.zone.now} #{message}"
```
Comment (matthewford) [inline app/services/performance_platform_service.rb]: Do not write to stdout. Use Rails' logger if you want to log.

**Comment by dreamfall** [comment]
:+1:

File: app/services/notifiers/email_notification_service.rb [hunk 5]
```diff
@@ -98,4 +102,10 @@ def winners_press_release_comments_request(award_year)
       end
     end
   end
+
+  class << self
+    def log_this(message)
+      p "[EmailNotificationService] #{Time.zone.now} #{message}"
```
Comment (matthewford) [inline app/services/notifiers/email_notification_service.rb]: Do not write to stdout. Use Rails' logger if you want to log.

File: app/services/performance_platform_service.rb [hunk 6]
```diff
@@ -199,4 +203,10 @@ def self.md5(string)
 
     md5.hexdigest
   end
+
+  class << self
+    def log_this(message)
+      p "[PerformancePlatformService] #{Time.zone.now} #{message}"
```
Comment (matthewford) [inline app/services/performance_platform_service.rb]: Do not write to stdout. Use Rails' logger if you want to log.

**Pull request merged**